src/middlewared/middlewared/plugins/etc.py

```diff
@@ -177,13 +184,16 @@ def __init__(self, *args, **kwargs):
             if renderer is None:
                 raise ValueError(f'Unknown type: {entry["type"]}')
 
+            self.args = args
```

Comment:
`continue` block above will skip resetting `self.args = tuple()`

With code like this it's better to do:

```
self.args = args
try:
    ...
finally:
    self.args = ()
```